### ISSUE TITLE
504 Gateway Timeout - Error AH01075

### ISSUE THREAD

**Issue by chrisTray** [dokuwiki/dokuwiki#4028]
Hello,

close to the time of upgrading from 2023-04-04 "Jack Jackrum" to 2023-04-04a "Jack Jackrum" requests to DokuWiki are non-reproducibly responded with "504 Gateway Timeout". Sometimes, I can call the login page but with loggin in it crashes. Then I can sometimes log in with a different browser. Maybe it is related to the authentication. DokuWiki is connected with an Active Directory.
I have since downgraded again, but no improvement.

The apache logs show the error:
AH01075: Error dispatching request to : (polling)

Otherwise there are no errors to be found.

System:
Oracle Linux 8
Apache 2.4
PHP 7.4.33

Thanks!
Chris

**Comment by fiwswe** [comment]
That does not sound like the problem has anything specifically to do with DokuWiki. Seems to be an issue with the hosting environment.

You might want to check the Apache access logs to see if any DW pages are actually being called.

Are you using mod_php or PHP_FPM? If the later then there might be PHP_FPM logs as well. Or maybe the issue is a problem with PHP_FPM where Apache can't dispatch there? (Just wild guesses…)

You could create a small test file such as the following in your DokuWiki directory and try to access it through the web browser or curl: `test.php`
```php
<?php
    echo 'Testing PHP execution: Version '.htmlspecialchars(phpversion()).' OK'.PHP_EOL;
?>
```
If that doesn't work then your problem is not caused by DokuWiki.

Side issue: PHP 7.4 is End-of-Life. Consider upgrading to PHP 8.2.

**Comment by chrisTray** [comment]
It's ridiculous. After opening this issue I upgraded dokuwiki back to 2023-04-04a "Jack Jackrum", updated the system packages and rebooted the system. This was not the first time I did it, but since that time, there was no new error. I will observe until next week.

And I know, there were to less information and to many possible error sources, but apache just showed a timeout and PHP_FPM had no errors in it's log files. It looked to me like dokuwiki has a long running process. And the surrounding environmend has not been changed.

Thanks so far.
